File: FMAplication/ClientApp/src/app/Shared/Enums/actionType.ts
import { MapObject } from './mapObject';

export enum ActionType {
    DistributionCheckProduct = 0,
    FacingCountProduct = 1,
    PlanogramCheckProduct = 2,
    PriceAuditProduct = 3
}

export class ActionTypeLabel{

    public static ActionType :  MapObject[] = [
    { id : 0, label : "Distribution Check" },
    { id : 1, label : "Facing Count" },
    { id : 2, label : "Planogram Check" },
    { id : 3, label : "Price Audit" }
    
    ];


}

File: FMAplication/Enumerations/ActionType.cs
using System.ComponentModel;

namespace FMAplication.Enumerations
{
    public enum ActionType
    {
        [Description("Distribution Check Product")]
        DistributionCheckProduct,
        [Description("Facing Count Product")]
        FacingCountProduct,
        [Description("Planogram Check Product")]
        PlanogramCheckProduct,
        [Description("Price Audit Product")]
        PriceAuditProduct
    }
}
